units/host_allocator.go

```diff
@@ -288,18 +288,29 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 	span := trace.SpanFromContext(ctx)
 	span.SetAttributes(
 		attribute.String(evergreen.DistroIDOtelAttribute, distro.Id),
+		attribute.String(fmt.Sprintf("%s.distro_provider", hostAllocatorAttributePrefix), distro.Provider),
+		attribute.String(fmt.Sprintf("%s.total_runtime", hostAllocatorAttributePrefix), distroQueueInfo.ExpectedDuration.String()),
+		attribute.String(fmt.Sprintf("%s.time_to_empty", hostAllocatorAttributePrefix), timeToEmpty.String()),
+		attribute.String(fmt.Sprintf("%s.time_to_empty_no_spawns", hostAllocatorAttributePrefix), timeToEmptyNoSpawns.String()),
+		attribute.Int(fmt.Sprintf("%s.distro_max_hosts", hostAllocatorAttributePrefix), distro.HostAllocatorSettings.MaximumHosts),
 		attribute.Int(fmt.Sprintf("%s.hosts_requested", hostAllocatorAttributePrefix), len(hostsSpawned)),
 		attribute.Int(fmt.Sprintf("%s.hosts_free", hostAllocatorAttributePrefix), nHostsFree),
 		attribute.Int(fmt.Sprintf("%s.hosts_running", hostAllocatorAttributePrefix), len(upHosts)),
+		attribute.Int(fmt.Sprintf("%s.hosts_total", hostAllocatorAttributePrefix), existingHosts.Stats().Total),
 		attribute.Int(fmt.Sprintf("%s.hosts_active", hostAllocatorAttributePrefix), existingHosts.Stats().Active),
 		attribute.Int(fmt.Sprintf("%s.hosts_idle", hostAllocatorAttributePrefix), existingHosts.Stats().Idle),
 		attribute.Int(fmt.Sprintf("%s.hosts_provisioning", hostAllocatorAttributePrefix), existingHosts.Stats().Provisioning),
 		attribute.Int(fmt.Sprintf("%s.hosts_quarantined", hostAllocatorAttributePrefix), existingHosts.Stats().Quarantined),
 		attribute.Int(fmt.Sprintf("%s.hosts_decommissioned", hostAllocatorAttributePrefix), existingHosts.Stats().Decommissioned),
 		attribute.Int(fmt.Sprintf("%s.task_queue_length", hostAllocatorAttributePrefix), distroQueueInfo.Length),
 		attribute.Int(fmt.Sprintf("%s.overdue_tasks", hostAllocatorAttributePrefix), distroQueueInfo.CountWaitOverThreshold),
+		attribute.Int(fmt.Sprintf("%s.overdue_tasks_in_groups", hostAllocatorAttributePrefix), totalOverdueInTaskGroups),
 		attribute.Float64(fmt.Sprintf("%s.seconds_to_empty", hostAllocatorAttributePrefix), timeToEmptyNoSpawns.Seconds()),
 		attribute.Float64(fmt.Sprintf("%s.queue_ratio", hostAllocatorAttributePrefix), float64(noSpawnsRatio)),
+		attribute.Float64(fmt.Sprintf("%s.host_queue_ratio", hostAllocatorAttributePrefix), float64(hostQueueRatio)),
+		attribute.Float64(fmt.Sprintf("%s.runtime_secs", hostAllocatorAttributePrefix), distroQueueInfo.ExpectedDuration.Seconds()),
+		attribute.Float64(fmt.Sprintf("%s.time_to_empty_mins", hostAllocatorAttributePrefix), timeToEmpty.Minutes()),
+		attribute.Float64(fmt.Sprintf("%s.time_to_empty_no_spawns_mins", hostAllocatorAttributePrefix), timeToEmptyNoSpawns.Minutes()),
```

Comment:
Do we need `time_to_empty_no_spawn_mins` when `seconds_to_empty` is already showing the same stats above on L308? Maybe we can rename `seconds_to_empty` to `time_to_empty_no_spawn_secs`.